### PR TITLE
2.x Prepare 2.0 release

### PR DESCRIPTION
**Pull request by gchtr**
Related:

- #2741

## Issue

Right before releasing 2.0, we should update the documentation.

## Solution

Prepare the changes.

## Impact

None.

## Usage Changes

None.

## Considerations

None.

## Testing

No.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,62 +52,48 @@ This is what Timber's `.twig` files look like (from this [Hello World example](h
...
-#### Who is it good for?
+### Who is it good for?
 Timber is great for any WordPress developer who cares about writing good, maintainable code. It helps teams of designers and developers working together. At [Upstatement](http://upstatement.com) we made Timber because while our entire team needs to participate in building WordPress sites, not everyone knows the ins-and-outs of the_loop(),  codex and PHP (nor should they). With Timber your best WordPress engineer can focus on building the `.php` files with requests from WordPress and pass the data into `.twig` files. Once there, designers can easily mark-up data and build out a site's look-and-feel.
 
-#### Related & Official Projects
+### Should I use it?
+
+Timber is MIT-licensed, so please use in personal or commercial work. Just don't re-sell it. Timber is used on [tens of thousands of sites](https://www.upstatement.com/timber/#showcase) (and tons more we don't know about)
+
+## Links
```
Comment (Levdbas) [inline README.md:87]: Links is still empty, what did you had in mind there?

File: README.md [hunk 2]
```diff
@@ -48,62 +52,48 @@ This is what Timber's `.twig` files look like (from this [Hello World example](h
 
 Once Timber is installed, it gives any WordPress theme the ability to take advantage of the power of Twig and other Timber features.
 
-### Looking for docs?
+## Documentation
+
 * [Timber Documentation](https://timber.github.io/docs/)
+* [Timber 2.0 Upgrade Guide](https://timber.github.io/docs/v2)
```
Comment (Levdbas) [inline README.md:58]:
```suggestion
* [Timber 2.0 Upgrade Guide](https://timber.github.io/docs/v2/upgrade-guides/2.0/)
```

Comment (Levdbas) [reply README.md:58]: Better to link to the actual guide.

Once v2 will go live, will the docs still live under the /v2/ slug? And do we need to prepare for that?

File: README.md (hunk 1 above)
Comment (gchtr) [reply README.md:87]: If you look at https://github.com/timber/timber/tree/2.x-prepare-release#links, you’ll see what I mean. I just put all the links for the related projects into a separate section with that "Link" heading.

**Comment by gchtr** [comment]
> Once v2 will go live, will the docs still live under the /v2/ slug? And do we need to prepare for that?

Yes, we still need to prepare for that. I mainly need to work on this in the https://github.com/timber/docs repository.

I thought about moving the /v2/ slug to the main docs path (https://timber-docs.test/docs/). But then, all existing links that specifically link to v2 documentation will not work anymore. We would have to create redirects for those (complicated). And also, all links to v1 docs wouldn’t lead to the intended page.

The solution I think would work better is to keep the version in the path. The [Laravel Documentation](https://laravel.com/docs) already does something similar (e.g. with a link `https://laravel.com/docs/10.x/installation`).

To make this work well, the following steps are needed:

- Make the homepage (https://timber-docs.test/docs/) show the v2 docs.
- Create https://timber-docs.test/docs/v1/ as an entry point for the v1 docs, but keep all v1 pages without /v1/ in the path.
- Show a banner in all v1 pages with an info that 2.x is the newer version. This could be similar to how Twig does this.
	CleanShot 2023-10-26 at 14 05 42

File: README.md (hunk 1 above)
Comment (Levdbas) [reply README.md:87]: Ah gotcha!

**Review by Levdbas — DISMISSED** [review]

**Comment by Levdbas** [comment]
> > Once v2 will go live, will the docs still live under the /v2/ slug? And do we need to prepare for that?
> 
> Yes, we still need to prepare for that. I mainly need to work on this in the https://github.com/timber/docs repository.
> 
> I thought about moving the /v2/ slug to the main docs path (https://timber-docs.test/docs/). But then, all existing links that specifically link to v2 documentation will not work anymore. We would have to create redirects for those (complicated). And also, all links to v1 docs wouldn’t lead to the intended page.
> 
> The solution I think would work better is to keep the version in the path. The [Laravel Documentation](https://laravel.com/docs) already does something similar (e.g. with a link `https://laravel.com/docs/10.x/installation`).
> 
> To make this work well, the following steps are needed:
> 
>     * Make the homepage (https://timber-docs.test/docs/) show the v2 docs.
> 
>     * Create https://timber-docs.test/docs/v1/ as an entry point for the v1 docs, but keep all v1 pages without /v1/ in the path.
> 
>     * Show a banner in all v1 pages with an info that 2.x is the newer version. This could be similar to how Twig does this.
>       CleanShot 2023-10-26 at 14 05 42

Let me know if you need some help with that @gchtr

**Comment by gchtr** [comment]
> Let me know if you need some help with that @gchtr

Thanks for offering your help. I will create a pull request in the docs repo that you can then test in a local environment.

**Comment by gchtr** [comment]
@Levdbas I’ve created a pull request in https://github.com/timber/docs/pull/80. I’d be happy if you could check that out!

**Comment by gchtr** [comment]
Another fix in https://github.com/timber/timber/pull/2829/commits/d7163f10f46853e679b130e48f6149ba9853cc28 to make the section archives work, see https://github.com/timber/docs/pull/80#issuecomment-1799145285.

**Review by Levdbas — APPROVED** [review]

**Pull request merged**